src/server/server.rs

```diff
@@ -33,7 +34,7 @@ use super::raft_client::RaftClient;
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 50;
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
-const MAX_GRPC_SEND_MSG_LEN: usize = 128 * 1024 * 1024;
+const MAX_GRPC_SEND_MSG_LEN: usize = 128 * REGION_SPLIT_SIZE as usize * 1024 * 1024;
```

Comment:
`REGION_SPLIT_SIZE` is already `256 * 1024 * 1024`. I think you use a huge length here.